server/app/config.js

```diff
@@ -105,31 +105,30 @@ app.use("/api", apiRouter);
 // 2. Ensure that the `reactBuildPath` points to the correct directory where your client's build artifacts are located.
 
 const reactBuildPath = path.join(__dirname, "/../../client/dist");
-const publicFolderPath = path.join(
-  __dirname,
-  "/../public/assets/images/upload"
-);
+const publicFolderPath = path.join(__dirname, "/../public");
```

Comment:
We have to be very careful with this change because it is used in a lot of pages - hopefully you checked that it does not break the other features

Reply:
It's good 👍